content/docs/15.how-to-guides/google-spreadsheets.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Google Spreadsheets
+icon: /docs/icons/tutorial.svg
+---
+
+Learn step-by-step how to read data from Google Spreadsheet in a Kestra flow.
+
+You can use any Google Spreadsheet for this tutorial. In case you do not have Google Spreadsheet, you can:
+
+1. Download the data from [this link](https://huggingface.co/datasets/kestra/datasets/raw/main/csv/orders.csv) and save it locally as `orders.csv` file.
+2. Create a new Google Spreadsheet.
+3. Navigate to the `File` menu on the top, and select `Import` option.
+4. Navigate to the `Upload` tab, and click on the `Browse` button.
+5. Select the recently created `orders.csv` file, and click on `Open` button at the buttom of the popup.
+6. On the `Import file` popup, choose the import location as `Replace spreadsheet` and separator type as `Detect automatically`. In this case, it does not matter whether you check or uncheck the box `Convert text to numbers, dates, and formulas`. Click on `Import data` button.
+7. The contents of the file will be populated in the spreadsheet.
+8. You can put an appropriate title to the spreadsheet, and name the sheet containing the order records as `orders`.
+
+![sheet_import](/docs/how-to-guides/google-spreadsheets/sheet_import.png)
+
+![sheet_upload](/docs/how-to-guides/google-spreadsheets/sheet_upload.png)
+
+![sheet_import_data](/docs/how-to-guides/google-spreadsheets/sheet_import_data.png)
+
+![sheet_uploaded_data](/docs/how-to-guides/google-spreadsheets/sheet_uploaded_data.png)
+
+Now that we have the spreadsheet ready, let us proceed to assign appropriate authorization for the spreadsheet. For this, 
+
+1. Go to the GCP console, navigate to the [IAM service](https://console.cloud.google.com/iam-admin/iam).
+2. Select `Service accounts` from the left navigation menu.
+3. Click on `Create Service Account` on the top. You may choose to use an existing service account in which case you can skip the next step.
+4. Put in appropriate service account name, service account id (the auto-populated value should be good to start with) and service account description, and click on `Done`.
+
+![create_service_account_1](/docs/how-to-guides/google-spreadsheets/create_service_account_1.png)
+
+![create_service_account_2](/docs/how-to-guides/google-spreadsheets/create_service_account_2.png)
+
+The new service account has been created. Lets add a key to the service account.
+
+1. Click on the corresponding service account from the Service Accounts page.
+2. Navigate to `Keys` tab, and click on `Add Key` -> `Create new key`.
+3. On the `Create private key` popup, select `JSON` option, and click on `Create`.
+4. This will download the service account JSON file on your local computer.
```

Comment:
Add a link to the service account guide we published recently too

Reply:
Done